File: pregunta 3/uno_f_superior.py
def fibonacci(n):
    a=1
    b=0
    for i in range(n):
        print(b)
        b = a+b
        a = b-a
        i += 1

def ordensuperior(funcion, n=5):
    funcion(n)
    


print("Ingrese un valor")
n = int(input())
print("La Serie de Fibonacci es: ")
ordensuperior(fibonacci,n)

File: pregunta 3/uno_f_superior.fs
open System

let fib n =
        let mutable a=1
        let mutable b=0
        for i in 1 .. n do
        printfn "  %d" b
        b <- a+b
        a <- b-a

let ordensuperior funcion n= 
    printfn "Serie de Fibonacci de %i terminos" n
    funcion n

[<EntryPoint>]
let main argv = 
    let n = 9
    ordensuperior fib n
        
    let tecla = Console.ReadKey()
    0 // devolver un código de salida entero